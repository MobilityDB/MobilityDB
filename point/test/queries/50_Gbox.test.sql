/*****************************************************************************
 * Gbox
 *****************************************************************************/

SELECT gbox 'GBOX((1.0, 2.0), (1.0, 2.0))';
SELECT gbox 'GBOX Z((1.0, 2.0, 3.0), (1.0, 2.0, 3.0))';
SELECT gbox 'GBOX M((1.0, 2.0, 3.0), (1.0, 2.0, 3.0))';
SELECT gbox 'GBOX ZM((1.0, 2.0, 3.0, 4.0), (1.0, 2.0, 3.0, 4.0))';
SELECT gbox 'GEODBOX((1.0, 2.0, 3.0), (1.0, 2.0, 3.0))';

SELECT gbox(1,2,3,4);
SELECT gbox(1,2,3,4,5,6);
SELECT gbox(1,2,3,4,5,6,7,8);
SELECT gbox3dm(1,2,3,4,5,6);
SELECT geodbox(1,2,3,4,5,6);

/*****************************************************************************/