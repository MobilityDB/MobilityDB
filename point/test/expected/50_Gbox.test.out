/*****************************************************************************
 * Gbox
 *****************************************************************************/
SELECT gbox 'GBOX((1.0, 2.0), (1.0, 2.0))';
       gbox        
-------------------
 GBOX((1,2),(1,2))
(1 row)

SELECT gbox 'GBOX Z((1.0, 2.0, 3.0), (1.0, 2.0, 3.0))';
          gbox           
-------------------------
 GBOX Z((1,2,3),(1,2,3))
(1 row)

SELECT gbox 'GBOX M((1.0, 2.0, 3.0), (1.0, 2.0, 3.0))';
          gbox           
-------------------------
 GBOX M((1,2,3),(1,2,3))
(1 row)

SELECT gbox 'GBOX ZM((1.0, 2.0, 3.0, 4.0), (1.0, 2.0, 3.0, 4.0))';
             gbox             
------------------------------
 GBOX ZM((1,2,3,4),(1,2,3,4))
(1 row)

SELECT gbox 'GEODBOX((1.0, 2.0, 3.0), (1.0, 2.0, 3.0))';
           gbox           
--------------------------
 GEODBOX((1,2,3),(1,2,3))
(1 row)

SELECT gbox(1,2,3,4);
       gbox        
-------------------
 GBOX((1,3),(2,4))
(1 row)

SELECT gbox(1,2,3,4,5,6);
          gbox           
-------------------------
 GBOX Z((1,3,5),(2,4,6))
(1 row)

SELECT gbox(1,2,3,4,5,6,7,8);
             gbox             
------------------------------
 GBOX ZM((1,3,5,7),(2,4,6,8))
(1 row)

SELECT gbox3dm(1,2,3,4,5,6);
         gbox3dm         
-------------------------
 GBOX M((1,3,5),(2,4,6))
(1 row)

SELECT geodbox(1,2,3,4,5,6);
         geodbox          
--------------------------
 GEODBOX((1,3,5),(2,4,6))
(1 row)

/*****************************************************************************/
